next.clayui.com/content/docs/css/components/icons.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Icons"
+description: "Icons are a visual representation of an idea and/or action."
+---
+
+<div class="nav-toc">
+
+- [Usage](#usage)
+- [Language Flags](#language-flags)
+- [SVG Icons as Background Images](#svg-icons-as-background-images)
+- [SVG-url()](#svg-url())
+- [Why do we use SVG?](#why-do-we-use-svg)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/icons/">Icons Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	See the implementation of the <a href="/docs/components/icons.html">Icons component in React</a> following the Lexicon <a href="https://liferay.design/lexicon/core-components/icons/">Icons Pattern</a>.
+</div>
+
+### Usage
+
+> We use SVG elements that link to an SVG sprite, like so:
```

Comment:
We no longer need to add these texts in blockquotes. The same for the others on this page.